appendices/devbook-guide/text.xml

```diff
@@ -40,137 +39,111 @@ tags used in a GuideXML document:
-section has a title.  Here's an example chapter with a single section,
-consisting of a paragraph.  If you append this XML to the XML in the <uri
-link="#doc_chap2_pre1">previous excerpt</uri> and append a
-<c>&lt;/guide&gt;</c> to the end of the file, you'll have a valid (if minimal)
-guide document:
+structural elements of the document. Chapters are divided into sections, each
+section can hold one or more subsections, and so on, down to the subsubsection
+level. Section elements at any level have a title. Here's an example section
+with a single subsection, consisting of a paragraph. If you append this XML to
+the XML in the <uri link="::appendices/devbook-guide/#Basic structure">previous
+excerpt</uri> and append <c>&lt;/chapter&gt;</c> and <c>&lt;/guide&gt;</c>
+to the end of the file, you'll have a valid (if minimal) guide document:
 </p>
 
 <codesample lang="sgml"><!-- Minimal guide example -->
-&lt;chapter&gt;
-&lt;title&gt;This is my chapter&lt;/title&gt;
 &lt;section&gt;
-&lt;title&gt;This is section one of my chapter&lt;/title&gt;
+&lt;title&gt;This is my section&lt;/title&gt;
+&lt;subsection&gt;
+&lt;title&gt;This is subsection one of my section&lt;/title&gt;
 &lt;body&gt;
 
 &lt;p&gt;
-This is the actual text content of my section.
+This is the actual text content of my subsection.
 &lt;/p&gt;
 
 &lt;/body&gt;
+&lt;/subsection&gt;
 &lt;/section&gt;
-&lt;/chapter&gt;
 </codesample>
 
 <p>
-Above, I set the chapter title by adding a child <c>&lt;title&gt;</c>
-element to the <c>&lt;chapter&gt;</c> element.  Then, I created a section by
-adding a <c>&lt;section&gt;</c> element.  If you look inside the
-<c>&lt;section&gt;</c> element, you'll see that it has two child elements <d/> a
-<c>&lt;title&gt;</c> and a <c>&lt;body&gt;</c>.  While the <c>&lt;title&gt;</c>
-is nothing new, the <c>&lt;body&gt;</c> is <d/> it contains the actual text
-content of this particular section.  We'll look at the tags that are allowed
-inside a <c>&lt;body&gt;</c> element in a bit. 
+Above, I set the section title by adding a child <c>&lt;title&gt;</c> element
+to the <c>&lt;section&gt;</c> element. Then, I created a subsection by adding
+a <c>&lt;subsection&gt;</c> element. If you look inside the
+<c>&lt;subsection&gt;</c> element, you'll see that it has two child elements
+<d/> a <c>&lt;title&gt;</c> and a <c>&lt;body&gt;</c>. While the
+<c>&lt;title&gt;</c> is nothing new, the <c>&lt;body&gt;</c> is <d/>
+it contains the actual text content of this particular subsection. We'll look
+at the tags that are allowed inside a <c>&lt;body&gt;</c> element in a bit.
 </p>
 
 <note>
-A <c>&lt;guide&gt;</c> element must contain at least one <c>&lt;chapter&gt;</c>
-elements, a <c>&lt;chapter&gt;</c> must contain at least one
-<c>&lt;section&gt;</c> elements and a <c>&lt;section&gt;</c> element must
-contain at least one <c>&lt;body&gt;</c> element.  
+Section elements at any level (i.e., <c>&lt;chapter&gt;</c>,
```

Comment:
Arguably, `<chapter/>` is not a section element. We can start the sentence with `The elements <c>&lt;chapter&gt;<c/>, <c>&lt;section&gt;<c/>, <c>&lt;subsection&gt;<c/>, and <c>&lt;subsubsection&gt;<c/> may contain a <c>&lt;body&gt;<c/>.` and so on.

Reply:
Done.